package-lock.json

```diff
@@ -1,8 +1,14011 @@
 {
   "name": "django-react-template",
   "version": "0.1.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
   "requires": true,
+  "packages": {
+    "": {
+      "name": "django-react-template",
+      "version": "0.1.0",
+      "dependencies": {
+        "bootstrap": "^5.1.1",
+        "django-react-loader": "^0.1.7",
+        "nwb": "^0.24.7",
+        "react": "^16.13.1",
+        "react-dom": "^16.13.1",
+        "webpack-bundle-tracker": "^1.4.0"
+      }
+    },
+    "node_modules/@babel/cli": {
```

Comment:
14,085 additions is kind of a lot for such a small change - I noticed that most of these lines come from packages that have names prepended with `node_modules/`, which leads me to suspect something funky happened when this lock file was generated.